doc/examples/applications/plot_cornea_spot_inpainting.py

```diff
@@ -166,7 +166,7 @@ def plot_comparison(plot1, plot2, title1, title2):
 
 footprint = ski.morphology.diamond(3)
 mask_open = ski.morphology.opening(mask_2, footprint)
-plot_comparison(mask_2, mask_open, "mask before", "after opening")
+plot_comparison(mask_2, mask_open, "Mask before", "After opening")
```

Comment:
Why not, but this is unrelated to the current issue, and the PR is very long already, so I would prefer having these changes in a separate/dedicated PR. Thanks!

Reply:
Reverted. In general, though, it would be helpful to know if plot titles should start with a capital letter or not. At present, there is a mix, even within the same example.

Reply:
You are absolutely right, @GParolini. It's just that, precisely, this issue deserves its own... issue/PR :wink: 

(see also @lagru's comment about avoiding blowing up the diff :pray:)